templates/includes/record-revealed.html

```diff
@@ -5,7 +5,7 @@
             <div class="col-md-8 col-sm-12">
                 <p class="explore-intro__title explore-intro__title--meta">Record revealed</p>
                 <h1 class="explore-intro__title">{{ page.title }}</h1>
-                <p class="explore-intro__paragraph">{{ page.standfirst }}</p>
+                <p class="explore-intro__paragraph">{{ page.intro }}</p>
```

Comment:
`page.intro` is RichText, for should have the `|richtext` filter (from `wagtailcore_tags`) applied to it. 

We'll also need to lose the hardcoded `<p>` tag, as the generated HTML will have it's own.

@William-Blackie What are the implications here on styling?

Reply:
Hi @ababic this should be done now! 👍